modules/mcagar/src/main/ts/ephox/mcagar/alien/EditorTypes.ts

```diff
@@ -1,5 +1,8 @@
 type EventCallback = (event: any) => void;
 
+export type GetContentFormatter = (editor: Editor, args: GetContentArgs) => any;
+export type SetContentFormatter = (editor: Editor, content: any, args: SetContentArgs) => any;
+
```

Comment:
```suggestion
export type GetContentFormatter = (editor: Editor, args: GetContentArgs) => any;
export type SetContentFormatter = (editor: Editor, content: any, args: SetContentArgs) => any;

```
This should be reverted as it's not being used anymore.